src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const promise1 = new Promise(function (resolve, reject) {
+  logo.addEventListener('click', () => resolve());
```

Comment:
Consider adding a rejection handler for `promise1` to handle potential errors during the event listener setup. This will help avoid unhandled promise rejections.